src/main/java/org/recordrobotics/charger/subsystems/Claw.java

```diff
@@ -33,20 +34,26 @@ public Claw() {
 		ShuffleboardTab tab = Shuffleboard.getTab(Constants.DATA_TAB);
 		_voltageEntry = tab.add("Motor Current", 0).getEntry();
 		_encoderEntry = tab.add("Encoder Value", 0).getEntry();
+
+		SmartDashboard.putNumber("Voltage Output", 0);
 	}
 
 	@Override
 	public void periodic() {
 		_voltageEntry.setDouble(_motor.getOutputCurrent());
 		_encoderEntry.setDouble(getPosition());
+		if (!getSwitchState()) {
+			resetEncoders();
```

Comment:
The first time we resetEncoders when we hit the limit switch, we should set a boolean flag. If that flag is false (when we never hit the limit switch in this session yet), we should disable grabbing. You can only release, till the limit switch is hit. Only then it's safe to grab, since we then know when to stop.